Classes/KIFUIViewTestActor.m

```diff
@@ -230,6 +230,14 @@ - (void)enterText:(NSString *)text;
     [self enterText:text expectedResult:nil];
 }
 
+- (void)setText:(NSString *)text;
```

Comment:
Can you move this method below the next method so the `enterText` methods stay together?